package.json

```diff
@@ -81,14 +82,29 @@
     "tape-catch": "^1.0.4",
     "testron": "^1.2.0",
     "tween.js": "^16.3.5",
-    "uglify-js": "^2.6.1"
+    "uglify-js": "^2.6.1",
+    "module-alias": "^2.0.0",
+    "pre-commit": "^1.2.2",
+    "react-dom": "^15.4.0",
+    "reify": "^0.4.4",
+    "tap-browser-color": "^0.1.2",
+    "tape": "^4.5.1",
+    "tape-catch": "^1.0.4",
+    "webpack": "^2.2.0-rc.3"
   },
   "peerDependencies": {
     "luma.gl": "3.0.0-alpha2",
     "react": "0.14.x - 15.x",
     "react-dom": "0.14.x - 15.x"
   },
   "scripts": {
+    "wp-start": "(cd examples/main && yarn && npm start)",
```

Comment:
> do we need yarn in the dep or devDep?

> Thus this line in deck.gl exhibits

Good question. While very clever, I personally think that the line in the exhibits looks much too intimidating. It makes a simple example and makes it feel complicated - and I think it should be removed. My goal is super simple package.json files that can be understood at a glance.

We could just require that yarn is installed for development of the library, and just make a note of it in the README?

Or we could require the user to go to examples/main and install any way he wants before running start.